frontend/src/scenes/web-analytics/WebAnalyticsLiveUserCount.tsx

```diff
@@ -0,0 +1,30 @@
+import { IconLive } from '@posthog/icons'
+import { useValues } from 'kea'
+import { Tooltip } from 'lib/lemon-ui/Tooltip'
+import { humanFriendlyLargeNumber, humanFriendlyNumber } from 'lib/utils'
+import { teamLogic } from 'scenes/teamLogic'
+import { liveEventsTableLogic } from 'scenes/web-analytics/liveWebAnalyticsLogic'
+
+export const WebAnalyticsLiveUserCount = (): JSX.Element | null => {
+    const { liveUserCount, liveUserUpdatedSecondsAgo } = useValues(liveEventsTableLogic)
+    const { currentTeam } = useValues(teamLogic)
+
+    if (liveUserCount == null || liveUserUpdatedSecondsAgo == null) {
+        return null
+    }
```

Comment:
It's a bit odd that this stat completely disappears when nobody's active (and the disappearance also causes a large layout shift). I think this should always be present if the flag's on.

Reply:
Ah right! I didn't realise that the api returned undefined for 0 users, I'll fix it

Reply:
Could add it as an action for the page like this:

<img width="1353" alt="Screenshot 2024-06-25 at 19 19 28" src="https://github.com/PostHog/posthog/assets/2056078/0bfc8203-eb0b-48f0-ae9a-82296e0f647b">